temporalcli/commands.server.go

```diff
@@ -101,11 +101,11 @@ func (t *TemporalServerStartDevCommand) run(cctx *CommandContext, args []string)
 	if friendlyIP == "127.0.0.1" {
 		friendlyIP = "localhost"
 	}
-	cctx.Printer.Printlnf("Temporal server is running at: %v:%v", friendlyIP, t.Port)
+	cctx.Printer.Printlnf("%-16s %v:%v", "Temporal server:", friendlyIP, t.Port)
```

Comment:
I'd also be on board with something even cleaner here such as "Temporal" if others like that.